notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<ul><li>So we need to write these additional codes all the time? --&gt; So do we need to write this boiler-plate preprocessing code each time?</li><li> Once we write a code, we can wrap the function with <code><a href="https://www.tensorflow.org/api_docs/python/tf/function?version=nightly" target="_blank">@tf.function()</a></code> decorator. --&gt; We should wrap our processing code with the @tf.function decorator.</li><li>The following sentence is a bit cryptic: "When saving a <code>tf.function</code>, you're really saving the <code>tf.function</code>'s cache of ConcreteFunctions." Is there a way to explain it in a simpler way?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Thank you. I modified the first two items.</p><p>I removed the sentence in the third item and added the official guide page as a reference.</p>